test/utilSpec.js

```diff
@@ -12,4 +13,31 @@ describe('Utils', function () {
         .throws(() => Crypto.aesDecryptObject('garbage', Buffer.from('trash', 'base64')))
     })
   })
+
+  describe('details', function ()  {
```

Comment:
The transport tests already use the class fully; the utils tests are just to catch the edge cases not covered by the other tests